public/components/header.jsx

```diff
@@ -1,20 +1,34 @@
+import { useEffect, useState } from "react";
 import css from "../css/header.module.css";
+import UserGreeting from "./userGreeting";
```

Comment:
have consistency with your imports order - in all other files currently the order is import from system (like react), then import components and the last one is css